webdriver/tests/bidi/storage/get_cookies/filter.py

```diff
@@ -43,7 +43,6 @@ async def test_filter(bidi_session, new_tab, test_page, origin, domain_value, fi
     recursive_compare(
         {
             "domain": domain_value(),
-            "expiry": None,
```

Comment:
Does this imply `expiry` should not be present, or it should be equal to `null`? The [`recursive_compare`](https://github.com/web-platform-tests/wpt/blob/2bc113a64c84abd08e4d570b81edd0a7cf0143ab/webdriver/tests/bidi/__init__.py#L21) requires it to be present and equal to `null`.

Same goes to other places in this file and in `webdriver/tests/bidi/storage/get_cookies/partition.py`

Reply:
From [the spec](https://w3c.github.io/webdriver-bidi/#type-network-Cookie) it looks like `expiry` should be equal `null` if it's not set. So my intention was to test that.

Reply:
ahhh, right, it was set to `null` at point 5 and then omitted at the point 10 🤦🏻‍♀️
I'll update it